utils/profiler.go

```diff
@@ -43,6 +47,9 @@ func StartProfile(cpuprofile, memprofile string) error {
 	}
 
 	if memprofile != "" {
+		go func() {
+			http.ListenAndServe("0.0.0.0:6060", nil)
```

Comment:
Please do not force import `net/http/pprof` in utils package, use `-metric-web` flag to start cqld/cql-minerd instead.